features/admin.userstores.v1/constants/user-store-constants.ts

```diff
@@ -48,6 +48,12 @@ export class UserStoreManagementConstants {
 
     // Name of the readonly DEFAULT userstore property.
     public static readonly USER_STORE_PROPERTY_READ_ONLY: string = "ReadOnly";
+
+    /**
+     * Set of keys used to enable/disable features.
+     */
+    public static readonly FEATURE_DICTIONARY: Map<string, string> = new Map<string, string>()
+        .set("USER_STORE_REMOTE", "userStores.type.remote");
```

Comment:
```suggestion
    public static readonly FEATURE_DICTIONARY: Map<string, string> = new Map([
	    ["USER_STORE_REMOTE", "userStores.type.remote"]
	]);
```
Use array-based Map instantiation, which enhances readability by removing multiple .set calls (when growing) and provides a cleaner, more scalable structure.
https://github.com/wso2/identity-apps/blob/35d513e17a4acd2e408a28caf1a953561a0db544/features/admin.applications.v1/constants/application-management.ts#L68-L84 this is a bad practice we have followed.